admin_guide/ipsec.adoc

```diff
@@ -54,24 +54,17 @@ After modifying the MTU in the {product-title} configuration, the SDN must be
 made aware of the change by removing the SDN interface and restarting the
 {product-title} node process.
 +
-.. Stop the node:
+.. Remove the SDN interface.
 +
 ----
-# systemctl stop atomic-openshift-node
-# ovs-vsctl del-br br0
-# systemctl start atomic-openshift-node
+# oc exec <SDN/OVS podname> -- ovs-vsctl del-br br0
 ----
 +
-.. Remove the SDN interface
+.. Restart SDN and OVS pods.
 +
 ----
-# ovs-vsctl del-br br0
-----
-+
-.. Restart the host to start the node:
-+
-----
-# reboot
+#oc delete pod -n openshift-sdn -l=app=ovs
```

Comment:
Add  spaces between the # character and each command. Also, # indicates that you need to run the command as root. Is that true here?

Reply:
done